src/auth/computeclient.ts

```diff
@@ -68,8 +74,11 @@ export class Compute extends OAuth2Client {
     try {
       // TODO: In 2.0, we should remove the ability to configure the tokenUrl,
       // and switch this over to use the gcp-metadata package instead.
-      res = await this.transporter.request<CredentialRequest>(
-          {url, headers: {'Metadata-Flavor': 'Google'}});
+      res = await ax.request<CredentialRequest>({
+        url,
+        headers: {'Metadata-Flavor': 'Google'},
+        raxConfig: {noResponseRetries: 8, retry: 8, instance: ax}
```

Comment:
8 times is a bit extreme. Python is 3, I believe.

Reply:
Hah, Cunningham's Law proves itself again